selfdrive/car/subaru/carstate.py

```diff
@@ -143,6 +150,7 @@ def update(self, cp, cp_cam):
     self.right_blinker_on = cp.vl["Dashlights"]['RIGHT_BLINKER'] == 1
     self.seatbelt_unlatched = cp.vl["Dashlights"]['SEATBELT_FL'] == 1
     self.steer_torque_driver = cp.vl["Steering_Torque"]['Steer_Torque_Sensor']
+    self.steer_torque_motor = cp.vl["Steering_Torque"]['Steer_Torque_Output']
```

Comment:
unused?

Reply:
Unused now, yes, I can remove it
Previously used when using the same steer limit as Toyota

Reply:
Unused now, yes, I can remove it
Previously used when using the same steer limit style as Toyota